sdk/sdk.go

```diff
@@ -214,6 +214,27 @@ func (s *IOSTDevSDK) GetAccountInfo(id string) (*rpcpb.Account, error) {
 	return value, nil
 }
 
+// GetTokenBalance return token balance
+func (s *IOSTDevSDK) GetTokenBalance(account string, token string) (*rpcpb.GetTokenBalanceResponse, error) {
+	if s.rpcConn == nil {
+		if err := s.Connect(); err != nil {
+			return nil, err
+		}
+		defer s.CloseConn()
+	}
+	client := rpcpb.NewApiServiceClient(s.rpcConn)
+	req := &rpcpb.GetTokenBalanceRequest{
+		Account:        account,
+		Token:          token,
+		ByLongestChain: true,
```

Comment:
It should be "ByLongestChain: s.useLongestChain,"

Reply:
> It should be "ByLongestChain: s.useLongestChain,"

fixed